config/parallel.json

```diff
@@ -0,0 +1,37 @@
+{
+    "capabilities": {
+        "bstack:options": {
+            "projectName": "Pytest Browserstack",
+            "buildName": "pytest-browserstack",
+            "sessionName": "parallel_test",
+            "local": false,
+            "debug" : "true",
+            "networkLogs": "true",
+            "seleniumVersion": "3.14.0"
```

Comment:
We can remove this, if not present the test will run on default selenium version.

Reply:
Done https://github.com/browserstack/pytest-browserstack/pull/2/commits/998555687793bd1c39a34317c2a5b3d86319041a